bne.lic

```diff
@@ -67,10 +85,67 @@ class BNE
         end
       end
     end
+
     bput('burgle', 'You make short work of the lock', 'You scale up the side of a wall')
-      #for each room_list  - tbd
-    fput('go window')
+
+    #stow entry method if needed once inside - easier loot managment
+    stow_hands
+
+    #TODO:  Define list of room types, and searchable objects to search for loot (https://elanthipedia.play.net/Breaking_and_Entering#Rooms)
+    #       Write movement and get out routines
+    if @bne_settings['search_count'] > 0
+      search_for_loot('counter')
+    end
+
+    bput('go window', 'You take a moment to reflect on the caper')
+
+  end
+
+  def search_for_loot(target)
+    if /It looks valuable/ =~ bput("search #{target}",'It looks valuable','Roundtime','I could not')
+      if @bne_settings['dispose']
+        drop_item("#{DRC.right_hand}")
+      else
+        put_item?("#{DRC.right_hand}")
+      end
+    end
+  end
+
+  def put_item?(item)
```

Comment:
💡 Why does `put_item?` return a boolean if its return value isn't used?

Reply:
Ripped it out of steal, for future use with bining/pawning.